examples/install_requirements.py

```diff
@@ -71,4 +72,19 @@
             print("Couldn't install dependencies as wheels and trying to compile from sources failed")
             print("Check https://github.com/luxonis/depthai-python#dependencies on retrieving dependencies for compiling from sources")
 
-subprocess.check_call([sys.executable, f"{examples_dir}/downloader/downloader.py", "--all", "--cache_dir", f"{examples_dir}/downloader/", "--num_attempts", "5", "-o", f"{examples_dir}/models"])
+resources = {
+    "models/mobilenet-ssd_openvino_2021.2_6shave.blob": "https://artifacts.luxonis.com/artifactory/luxonis-depthai-data-local/network/mobilenet-ssd_openvino_2021.2_6shave.blob",
+    "models/mobilenet-ssd_openvino_2021.2_5shave.blob": "https://artifacts.luxonis.com/artifactory/luxonis-depthai-data-local/network/mobilenet-ssd_openvino_2021.2_5shave.blob",
+    "models/mobilenet-ssd_openvino_2021.2_8shave.blob": "https://artifacts.luxonis.com/artifactory/luxonis-depthai-data-local/network/mobilenet-ssd_openvino_2021.2_8shave.blob",
+    "models/tiny-yolo-v3_openvino_2021.2_6shave.blob": "https://artifacts.luxonis.com/artifactory/luxonis-depthai-data-local/network/tiny-yolo-v3_openvino_2021.2_6shave.blob",
+    "models/tiny-yolo-v4_openvino_2021.2_6shave.blob": "https://artifacts.luxonis.com/artifactory/luxonis-depthai-data-local/network/tiny-yolo-v4_openvino_2021.2_6shave.blob",
+    "construction_vest.mp4": "https://artifacts.luxonis.com/artifactory/luxonis-depthai-data-local/network/construction_vest.mp4"
+}
+import requests
+for path, url in resources.items():
```

Comment:
The main advantage of model downloader was caching, this approach lacks caching and downloads it every time.

Reply:
Yes having caching is key.  Otherwise it will be really annoying for users and also put like a 500x load on our servers.